backend/src/main/java/com/bakdata/conquery/io/storage/ModificationShieldedWorkerStorage.java

```diff
@@ -31,6 +33,10 @@ public Validator getValidator() {
 		return delegate.getValidator();
 	}
 
+	public Set<ManagedExecutionId> getCancelledQueries() {
+		return delegate.getWorker().getCancelled();
+	}
+
```

Comment:
Das macht von außen den Eindruck, dass hier etwas persistiert wurde

Reply:
ja, das ist leider ein bisschen dem Interface geschuldet. Ich kann aber noch schauen, wo ich die sonst noch platzieren kann. Aber macht CancelledQueries an und für sich in WorkerInformation Sinn? Wo ich so überlege sollte es eigentlich im QueryExecutor landen

Reply:
Ist im QueryExecutor